src/experiment_prototype/interface_classes/sequences.py

```diff
@@ -158,6 +157,7 @@ def build_sequence_pulses(self):
         for stage in self.decimation_scheme.stages:
             dm_rate *= stage.dm_rate
 
+        self.output_rx_rate = self.transmit_metadata["rxrate"] / dm_rate
```

Comment:
I think we can probably get this from the last `self.decimation_scheme.stages`, actually. That would be cleaner